### PR TITLE
New package: n-9.1.0

### PR DESCRIPTION
**Pull request by akierig**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

### CODE REVIEW

**Comment by akierig** [comment]
While not compiled per se, it is packaged by a number of package systems (including [MacPorts](https://ports.macports.org/port/n/details/) where I assumed maintainership a few months ago. it's analogous to `rbenv` which we do also package.

File: srcpkgs/n/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+short_desc="Simple command line NodeJS version management"
+maintainer="anelki <akierig@fastmail.de>"
+license="MIT"
+homepage="https://github.com/tj/n"
+changelog="https://raw.githubusercontent.com/tj/n/master/CHANGELOG.md"
+distfiles="https://github.com/tj/n/archive/refs/tags/v${version}.tar.gz"
+checksum=48306496413c61d37eeaa0a7328a4520b1da0c42739e046f6f9242de0d0ae270
+
+do_install() {
+	vbin bin/n n
```
Comment (mhmdanas) [inline srcpkgs/n/template:16]:
```suggestion
	vbin bin/n
```

Comment (paper42) [reply srcpkgs/n/template:16]: The makefile doesn't seem to respect $DESTDIR which means do_install would put it to the masterdir and not destdir.

Comment (paper42) [inline srcpkgs/n/template:16]:
```suggestion
	vbin bin/n n
	vdoc ${FILESDIR}/README.voidlinux
```

**Pull request merged**